runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java

```diff
@@ -52,4 +52,15 @@ default int pauseSkillAfter()
 	{
 		return 0;
 	}
+
+	@ConfigItem(
+		position = 2,
+		keyName = "trackSkillAfter99",
+		name = "Track skill after level 99",
+		description = "Configures whether or not skills will be tracked after level 99"
+	)
+	default boolean trackSkillAfter99()
```

Comment:
I would inverse this, e.g rename to Do not tracker after 99, and make it disabled by default